uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/modules/app-dashboard.js

```diff
@@ -202,37 +161,24 @@ define([
                                 return deferred.promise;
                             }],
                             taskList: ['$q', 'TaskService', function ($q, TaskService) {
-                                var deferred = $q.defer();
-
-                                $q.all([
-                                    TaskService.get({
-                                        'sequential': 0,
-                                        'assignee_contact_id': config.LOGGED_IN_CONTACT_ID,
-                                        'status_id': {
-                                            'NOT IN': config.status.resolve.TASK
-                                        }
-                                    }),
-                                    TaskService.get({
-                                        'sequential': 0,
-                                        'source_contact_id': config.LOGGED_IN_CONTACT_ID,
-                                        'status_id': {
-                                            'NOT IN': config.status.resolve.TASK
-                                        }
-                                    })
-                                ]).then(function (results) {
-                                    var taskId, taskList = [];
-
-                                    angular.extend(results[0], results[1]);
-
-                                    for (taskId in results[0]) {
-                                        taskList.push(results[0][taskId]);
-                                    }
-
-                                    deferred.resolve(taskList);
-
-                                });
-
-                                return deferred.promise;
+                              var deferred = $q.defer();
+
+                              $q.all([
+                                TaskService.get({
+                                  'sequential': 0,
+                                  'assignee_contact_id': config.LOGGED_IN_CONTACT_ID,
+                                  'status_id': { 'NOT IN': config.status.resolve.TASK }
+                                }),
+                                TaskService.get({
+                                  'sequential': 0,
+                                  'source_contact_id': config.LOGGED_IN_CONTACT_ID,
+                                  'status_id': { 'NOT IN': config.status.resolve.TASK }
+                                })
+                              ]).then(function (results) {
+                                deferred.resolve(_(results[0]).assign(results[1]).values().value());
```

Comment:
Can we simplify this part?
`_(results[0]).assign(results[1]).values().value()` 
Something like;
`var value = results[1]).values().value();
_(results[0]).assign(value)`

Just to make things more readable.